src/util/chaintype.cpp

```diff
@@ -41,3 +41,20 @@ std::optional<ChainType> ChainTypeFromString(std::string_view chain)
         return std::nullopt;
     }
 }
+
+std::string ChainTypeToDisplayString(ChainType chain)
+{
+    switch (chain) {
+    case ChainType::MAIN:
+        return "Bitcoin";
```

Comment:
Confirming - Is using sentence case for mainnet correct? Rest of them use lower case.

Reply:
Yeah, it is kinda' weird

Reply:
In 055dbaed69a2577b967fae3d9b7646ef3b01a254 _Added chaintype user-facing string display_:

This is used as follows in later commits:

> `Invalid Base58 %s address.`

In the original code it would not specify the network:

> `Invalid or unsupported Base58-encoded address.`

I don't think it's very useful to add the network name to address. Certainly not for mainnet, because most users have no idea (and no need to know) about test networks.

My suggestion would be drop this helper entirely.